internal/dag/dag.go

```diff
@@ -365,11 +384,14 @@ type Route struct {
 	// If false, traffic is allowed only if it doesn't match any rule.
 	IPFilterAllow bool
 
-	// IPFilterRules is a list of ipv4/6 filter rules for which matching
+	// IPFilterRules i /6 filter rules for which matching
 	// requests should be filtered. The behavior of the filters is governed
 	// by IPFilterAllow.
 	IPFilterRules []IPFilterRule
 
+	//
```

Comment:
can we condense these here as well 
left a comment on your design 
https://github.com/projectcontour/contour/pull/5866#discussion_r1540786806

Reply:
PTAL the updated design.